### ISSUE TITLE
Stuck after update

### ISSUE THREAD

**Issue by awaregit** [topjohnwu/Magisk#4200]
Device: SM-N950F
Android version: 9
Magisk version name: com.topjohnwu.magisk
Magisk version code: 22,1 (22100)

Well I believe this is an alien thing going on.
Magisk was working fine.
I just had a notification to update Magisk in the notifications bar.
I simply accepted, something showed downloading and then nothing happened.
So I just then went to check Magisk. I opened it and I chose to install latest 22.1 (22100) (20) App.
Well it downloads, I install it succesfully but then Magisk manager (already hidden and renamed) doesn't open.
It just shows a message "Magisk Manager: Upgrade to full Magisk Manager to finish the setup. Download and install?" I press OK , it downloads for some seconds something and then it shows a toast message "Please manually re-launch the app".
I re-launch Magisk Manager and get again "Magisk Manager: Upgrade to full Magisk Manager to finish the setup. Download and install?"...
A continuous loop in other words with no successful outcome.
Now:

Some apps that require root still work
A banking app is still working, another sees rooted and stops
Rootchecker app shows "root access is not properly installed"
Root check app shows "rooted"
In other words a big fat mess!
I strongly fear that If I reboot the device I will lose all remaning root capabilities. Correct???
I think of flashing with TWRP "Magick 22.zip" but then this may mean fresh rooting and ALL data will be lost. Correct???
Thank you all in advance, God bless you!
